src/runtime/CMakeLists.txt

```diff
@@ -154,6 +156,7 @@ set(RUNTIME_CXX_FLAGS
     # Necessary for using virtual functions in the runtime code.
     -fno-rtti
     -Wall
+    -Wc++20-designator
```

Comment:
Does this mean C++20 is required for our runtime now?  IIRC, we have so far stuck with C++17 or earlier.

Reply:
No, this is adding the warning to make sure we don't accidentally use C++20 designators (which I had done earlier in the PR).